Tests/xcodeprojMeasurementsTests/XcodeProjGenerator.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+import PathKit
+@testable import xcodeproj
+
+final class XcodeProjGenerator {
+    
+    struct GenerateSetting {
+        let targetCount: Int = 5
+        let groupCountInLevel: Int = 4
+        let fileCountInLevel: Int = 10
+        let maxGroupDepth: Int = 4
+    }
+    
+    @discardableResult
+    func generate(path: Path, sourceRoot: Path, setting: GenerateSetting = GenerateSetting()) throws -> XcodeProj {
```

Comment:
I think generation is much more flexible and better for performance tests than downloading a finished project.

Reply:
I agree with you here. It's more reliable as well since we don't need to pull something from the internet anymore.